packages/experimental/Button/src/Button.types.ts

```diff
@@ -111,16 +111,62 @@ export interface ButtonProps extends Omit<IWithPressableOptions<ViewProps>, 'onP
   fluid?: boolean;
 }
 
+export interface ButtonMacProps {
+  /*
+   * Name of the icon to show on the Button.
+   */
+  image?: Image;
+  /*
+   * Text to show on the Button.
+   */
+  title?: string;
+  /*
+   * ButtonStyle wrapped enum
+   */
+  buttonStyle?: ButtonMacStyle;
```

Comment:
I'm not sure if @chripear 's change in FluentUI apple means we would have the same styles across native and React Native, but I would rather not define a separate type purely for Mac Styles. At the very least, I would prefer that this is just `ButtonStyle` and the platform check happens inside `ButtonStyle`. 
But again, I'd rather we just have the same name for the styles across platforms, or map the RN styles to the native styles we have

Reply:
Agreed, we now have one buttonStyle for all platforms.